app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ErrorsController < ActionController::Base
+  include PrxAuth::Rails::Controller
+
+  layout false
+  skip_forgery_protection
```

Comment:
Prevents an additional `ActionController::InvalidAuthenticityToken` when someone POSTs to a non-existent route.